core/driver/sources/src/main/resources/reference.conf

```diff
@@ -125,5 +125,13 @@ probe {
     // treat all projects/repositories under the `trusted` directory as "safe" and will run the code without alerts.
     // By default, `trusted` is resolved as the root directory: "/".
     # trusted = "/"
+    //
+    // `diagnostics` is a directory where diagnostics data, like logs and cache from the intelliJ instance, can
```

Comment:
Is cache also going to be included/is it going to be useful?

Reply:
Hm, what cache directory? I did not find such directory inside the IntelliJ instance dir:
```
lkontowski@lkontowski-MacBook-Pro /Users/lkontowski 
% ls -1 /Applications/IntelliJ\ IDEA\ CE.app/Contents
CodeResources
Info.plist
MacOS
Resources
_CodeSignature
bin
jbr
lib
license
plugins
```

Reply:
I mean... it's mentioned in your comment ;)

Reply:
Oh okay, removed